pages/about.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../bootstrap-5.3.0-alpha1/dist/css/bootstrap.min.css">
+    <script src="../bootstrap-5.3.0-alpha1/dist/js/bootstrap.bundle.min.js"></script>
+    <link rel="stylesheet" href="../style.css">
+    <link rel="icon" href="../icons/airplane.svg">
+    <title>Travelly | About</title>
+    <meta name="description" content="Have a safe travel with us.">
+</head>
+<body>
+    <!-- --------------- section : intro -------------  -->
+    <section class="intro p-3">
+        <header class="main-head">
+            <button class="humberger navbar-toggler postion-fixed mt-3 toggler-example " type="button">
+                <span></span>
+                <span></span>
+                <span></span>
+            </button>
+        </header>
+        <div class="text text-center">
+            <h3 class="py-2 my-2 ">"Hello! Sharing world"</h3>
+            <h1 class="fw-semibold normal p-1 my-2 mb-4">CREATIVE COMMONS <br>
+                GLOBAL SUMMIT 2015
+            </h1>
+            <p class="border text-center w-100 bg-white text-black my-3 py-4 px-2 fs-2">
```

Comment:
Great job @Ali-0111 🤝

- [ ] Your paragraphs on the mobile view of the about page seem to be distorted and they are not replicating the given design properly. You may kindly fix all the text and fonts of the About section of your mobile version and resubmit for review.
- Your design
![Your-About](https://user-images.githubusercontent.com/63917725/219651249-83afec17-3b8d-4246-a123-d45d61a5044a.PNG)
- Given Design
![Given-About](https://user-images.githubusercontent.com/63917725/219651328-6f6c3a54-cf6f-4cd0-9960-dcc75307f77c.PNG)